apps/api/src/app/workflows-v2/workflow.controller.e2e.ts

```diff
@@ -319,8 +310,7 @@ describe('Workflow Controller E2E API Testing', () => {
       // Verify updated steps
       expect(prodWorkflowUpdated.steps).to.have.lengthOf(2);
       expect(prodWorkflowUpdated.steps[0].name).to.equal('Updated Email Step');
-      // TODO: verify that the stepId (or some) is the same across env for the same prod/dev step
-      expect(prodWorkflowUpdated.steps[0]._id).to.not.equal(prodWorkflowCreated.steps[0]._id);
+      expect(prodWorkflowUpdated.steps[0].stepId).to.equal(prodWorkflowCreated.steps[0].stepId);
```

Comment:
@ChmaraX, I was not sure why this test started to fail on this PR. Should the stepId and _id be the same in both cases?

Reply:
Do you mean to add _id is equal? if this is the case, i agree i forgot to add it. will do now

Reply:
done